shared/contracts/BaseTemplate.sol

```diff
@@ -210,7 +248,11 @@ contract BaseTemplate is APMNamehash, IsContract {
         _acl.createPermission(_settingsManager, _payroll, _payroll.MANAGE_ALLOWED_TOKENS_ROLE(), _permissionsManager);
     }
 
-    function _unwrapPayrollSettings(uint256[4] memory _payrollSettings) internal pure returns (address denominationToken, IFeed priceFeed, uint64 rateExpiryTime, address employeeManager) {
+    function _unwrapPayrollSettings(
+        uint256[4] memory _payrollSettings
+    )
```

Comment:
are these kind of breakdowns necessary?

Reply:
Yeah, to fulfill the rule of not having lines with more than 120 chars. We could of course remove the rule instead.

Reply:
Perhaps we can set it to something else. I'm pretty sure we don't follow this rule in other contract repos (since a lot of them have lines that go over 120 char), but can't find the configuration.